### PR TITLE
Zookeeper plugin

### PR DESCRIPTION
**Pull request by jrxFive**
Created a zookeeper plugin that fetches from the ‘mntr’ command will
output measurements that are int and string based

### CODE REVIEW

File: plugins/zookeeper/zookeeper_test.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import (
+	"testing"
+
+	"github.com/influxdb/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMemcachedGeneratesMetrics(t *testing.T) {
```
Comment (sparrc) [inline plugins/zookeeper/zookeeper_test.go]: name of this function shouldn't be memcached?

**Comment by sparrc** [comment]
Your method of unit testing requires that you add a zookeeper docker image to the `scripts/docker-compose.yml` file

**Comment by sparrc** [comment]
please write a README detailing all measurements that will be output, you can use the CPU_README.md file in the `plugins/system` directory as an example

File: plugins/zookeeper/zookeeper_test.go (hunk 1 above)
Comment (jrxFive) [inline plugins/zookeeper/zookeeper_test.go]: Whoops will fix that was basing the test file based off memcached test plugin.

File: plugins/zookeeper/zookeeper_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	z := &Zookeeper{
+		Servers: []string{testutil.GetLocalHost()},
+	}
+
+	var acc testutil.Accumulator
+
+	err := z.Gather(&acc)
+	require.NoError(t, err)
+
+	intMetrics := []string{"zookeeper_avg_latency", "zookeeper_packets_sent", "zookeeper_znode_count", "zookeeper_open_file_descriptor_count", "zookeeper_max_file_descriptor_count"}
```
Comment (sparrc) [inline plugins/zookeeper/zookeeper_test.go]: Line length ;-)

Comment (sparrc) [inline plugins/zookeeper/zookeeper_test.go]: please limit to 80 chars

**Comment by sparrc** [comment]
thank you @jrxFive! Just address the comments I made and I can merge :+1:

**Pull request closed without merging**